cumulusci/cumulusci.yml

```diff
@@ -1042,6 +1042,7 @@ flows:
           ancestor_id: latest_github_release
           version_base: latest_github_release
           version_type: minor
+          tag_prefix: $project_config.project__git__prefix_beta
```

Comment:
@Br4nd0r The tag_prefix option is on the wrong step and I didn't notice :( It's an option for github_release.